packages/clerk-js/src/core/resources/SignIn.ts

```diff
@@ -170,8 +170,12 @@ export class SignIn extends BaseResource implements SignInResource {
     });
   };
 
-  public authenticateWithRedirect = async (params: AuthenticateWithRedirectParams): Promise<void> => {
-    const { strategy, redirectUrl, redirectUrlComplete, identifier } = params || {};
+  public authenticateWithRedirect = async (
+    params: AuthenticateWithRedirectParams & {
+      unsafeMetadata?: SignUpUnsafeMetadata;
```

Comment:
⛔  the `unsafeMetadata` is only used and allowed to be passed in`sign_ups` (which is already supported) not `sign_ins`. You could also set `unsafeMetadata` via `User#update()` but not in sign_ins.
Did you find any reference about it in our docs? How did you end up with suggesting this change? What is your use case?

Reply:
I noticed that if a user tries to use the social button to log in without having created an account before, a new account will be created automatically (which I think is acceptable). However, I am currently relying on the unsafe metadata to perform certain actions in my backend. This functionality is currently only available for the Signup component, but I believe it should also be available for the SignIn component to cover this use case.